selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/CodenvyEditor.java

```diff
@@ -507,17 +525,35 @@ public void setCursorToLine(int positionLine) {
     loader.waitOnClosed();
     askForValueDialog.clickOkBtn();
     askForValueDialog.waitFormToClose();
-    waitActiveEditor();
+    waitActive();
     expectedNumberOfActiveLine(positionLine);
   }
 
   /**
-   * set cursor in specified position
+   * set cursor in specified position for current visible editor
    *
    * @param positionLine is the specified number line
    * @param positionChar is the specified number char
    */
-  public void setCursorToDefinedLineAndChar(int positionLine, int positionChar) {
+  public void setCursorToDefinedLineAndCharForVisible(int positionLine, int positionChar) {
+    openGoToLineFormAndSetCursorToPosition(positionLine, positionChar);
+    waitActive();
+    waitSpecifiedValueForLineAndChar(positionLine, positionChar);
+  }
+
+  /**
+   * set cursor in specified position for current active and focused editor
+   *
+   * @param positionLine is the specified number line
+   * @param positionChar is the specified number char
+   */
+  public void setCursorToDefinedLineAndCharForCurrentActive(int positionLine, int positionChar) {
```

Comment:
Method name setCursorToDefinedLineAndCharForCurrentActive() looks still to long, IMHO.
Can we use here goToPosition()?